cmd/buildah/list.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	listFlags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "quiet",
+			Name:  "quiet, q",
```

Comment:
Yes, I'd like to add an "omit" option here that would keep the table headers from printing when used.  "quiet" would be used to show just the image id's, much like docker does.  My main confusion here is I'd thought Nalin had already done this, but perhaps it wasn't merged?